tests/core/calendar_event_created/user_created_calendar_event/statements.json

```diff
@@ -0,0 +1,61 @@
+[
+   {
+      "actor" : {
+         "account" : {
+            "homePage" : "http://www.example.org",
+            "name" : 1
+         },
+         "name" : "event_creator_firstname event_creator_lastname"
+      },
+      "context" : {
+         "contextActivities" : {
+            "category" : {
+               "definition" : {
+                  "name" : {
+                     "en" : "EDLM Moodle LMS"
+                  },
+                  "type" : "http://id.tincanapi.com/activitytype/lms"
+               },
+               "id" : "http://www.example.org",
+               "objectType" : "Activity"
+            },
+            "parent" : {
+               "definition" : {
+                  "description" : {
+                     "en" : "test_course_summary"
+                  },
+                  "name" : {
+                     "en" : "test_course_name"
+                  },
+                  "type" : "https://w3id.org/xapi/cmi5/activitytype/course"
+               },
+               "id" : "http://www.example.org/course/view.php?id=1",
+               "objectType" : "Activity"
+            }
+         },
```

Comment:
So I might have missed this on the last PR and now need to go back and check, but this is invalid xAPI. The properties of the contextActivities object are arrays